### ISSUE TITLE
Is the plugin executable from cmd line without need of maven?

### ISSUE THREAD

**Issue by Thomasator** [krasa/krasa-jaxb-tools#75]
Hi,

I've tried your maven plugin to convert xsd to POJO with 303 validation annotation and its exactly what I was looking for. But I would like to be able to run the maven plugin from the cmd line without need of maven :D. I know that it sounds weird, but for my use I cant use maven rn. Is there a way to somehow run the plugin like jar?

Thx for the help
Thomasator

**Comment by krasa** [comment]
Nope. Unless you want to figure out how to package it and make a main class...
